src/TopoPlots.jl

```diff
@@ -35,4 +36,15 @@ export CloughTocher, SplineInterpolator, DelaunayMesh, NullInterpolator, Scatter
 # Extrapolators
 export GeomExtrapolation, NullExtrapolation
 
+@setup_workload begin
+    # Putting some things in `@setup_workload` instead of `@compile_workload` can reduce the size of the
+    # precompile file and potentially make loading faster.
+    data, positions = TopoPlots.example_data()
+    @compile_workload begin
+        # all calls in this block will be precompiled, regardless of whether
+        # they belong to your package or not (on Julia 1.8 and higher)
+        eeg_topoplot(view(data, :, 340, 1); positions)
```

Comment:
I dont know much about precompilation - but if you input a view, is the call to a view precompiled, or also to a Matrix?

Reply:
It's compiled for a view, which may or may not make a big difference, depending on what compilation gets triggered.

Reply:
we can go ahead and add a precompilation workload for both view and non view. If there's a huge amount of overlap, then that won't really increase precompile times. If there isn't much overlap, then it will increase precompile times but makes it more worthwhile to have both in the precompile cache.